patches/patch_aufs.bash

```diff
@@ -9,7 +9,7 @@ set -e
 # Target Kernel Version
 VERSION=$(grep '^VERSION\s*=' Makefile | cut -d= -f2 | sed 's/\s//g')
 PATCHLEVEL=$(grep '^PATCHLEVEL\s*=' Makefile | cut -d= -f2 | sed 's/\s//g')
-KVER=$VERSION.$PATCHLEVEL
+KVER=4.x-rcN
```

Comment:
This hardcode won't works on non 4-x kernel, can you make it dynamic and hardcode 4.x-rcN only in the 4.x-docker/include.mk, i.e: https://github.com/scaleway/kernel-tools/blob/master/3.2.34-apparmor/include.mk